plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchConnectorFactory.java

```diff
@@ -27,13 +29,24 @@
 import org.weakref.jmx.guice.MBeanModule;
 
 import java.util.Map;
+import java.util.Optional;
 
 import static java.util.Objects.requireNonNull;
 
 public class ElasticsearchConnectorFactory
         implements ConnectorFactory
 {
-    ElasticsearchConnectorFactory() {}
+    private final Optional<Module> extension;
+
+    ElasticsearchConnectorFactory()
```

Comment:
why not public?

Reply:
This was not used externally, but with the new refactors I made it public.